File: Programmers/level1/insufficientAmount/insufficientAmount.py
# 문제 출처 : https://programmers.co.kr/learn/courses/30/lessons/82612

def solution(price, money, count):

    need = 0
    for i in range(1, count+1):
        need += price*i
    print(need)
    if need <= money:
        return 0
    return need - money

File: Programmers/level1/insufficientAmount/insufficientAmount.js
// https://programmers.co.kr/learn/courses/30/lessons/82612?language=javascript

export const solution = (price, money, count) => {
    let answer = 0;
    let cnt = 1;
    while(cnt < count+1){
        let pri = price * cnt;
        answer += pri;
        cnt += 1;
    };
    if(answer - money > 0){
        return answer - money;
    } else {
        return 0;
    };
};